LICENSE

```diff
@@ -1,21 +1,14 @@
-MIT License
+Copyright (c) 2019 Sentry (https://sentry.io) and individual contributors. All rights reserved.
```

Comment:
Not sure, but do we need to update the year here to 2023? 🤔

Reply:
Good catch, thanks!